rhel/SPECS/grafana-db-migrator.spec

```diff
@@ -0,0 +1,33 @@
+%undefine _missing_build_ids_terminate_build
+
+%global release_hash b61138937c3a25c49be5548150a9df7e23f74d27
+
+Name:           grafana-db-migrator
+Version:        0.0.1
+Release:	    1%{?dist}
```

Comment:
nit: formatting quirk here

Reply:
heh, I had tabs instead of whitespaces and it looks good in VSCode :)